Models/model.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
```

Comment:
Can you confirm that it's connected to the mongo?

The connection code should not be in there. Should be in the apps entry point file, index.js or whatever.